data/json/recipes/weapon/bashing.json

```
@@ -594,16 +594,43 @@
     "type": "recipe",
     "activity_level": "MODERATE_EXERCISE",
     "result": "bo",
+    "id_suffix": "from long stick",
     "category": "CC_WEAPON",
     "subcategory": "CSC_WEAPON_BASHING",
     "skill_used": "fabrication",
-    "difficulty": 5,
+    "difficulty": 2,
+    "//": "Originator set difficulty to 5.  It's a carved, polished and oiled stick.  If you look at other recipes of that general difficulty, they're all way more difficult crafting.",
     "time": "5 h",
     "autolearn": true,
     "qualities": [ { "id": "SAW_W", "level": 1 }, { "id": "CUT", "level": 2 }, { "id": "FILE", "level": 1 } ],
     "proficiencies": [ { "proficiency": "prof_carving", "fail_multiplier": 1.5 } ],
     "components": [
-      [ [ "stick_long", 1 ], [ "q_staff", 1 ] ],
+      [ [ "stick_long", 1 ] ],
+      [
+        [ "any_tallow", 2, "LIST" ],
+        [ "cooking_oil", 16 ],
+        [ "cooking_oil2", 16 ],
+        [ "lamp_oil", 250 ],
+        [ "motor_oil", 250 ]
+      ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "bo",
+    "id_suffix": "from quarterstaff",
+    "//": "Significantly easier to make from a quarterstaff.",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_BASHING",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "270 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 2 }, { "id": "FILE", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_carving" } ],
```

You should not be able to bypass the saw requirement or fail multiplier by making it from a quarterstaff rather than a long stick, if you want to remove the saw requirement from here, then it should be removed from the other recipe too, or added to the quarterstaff recipe (Which would make more sense).
```suggestion
    "qualities": [ { "id": "SAW_W", "level": 1 }, { "id": "CUT", "level": 2 }, { "id": "FILE", "level": 1 } ],
    "proficiencies": [ { "proficiency": "prof_carving", "fail_multiplier": 1.5 } ],
```

---

The fail multiplier is similar to the failure rate from two rounds of crafting, each with a failure rate.
I might lower it, but I don't think keeping it the same makes sense.

I was on the fence with the saw requirement anyway. I am leaning towards removing it, but welcome suggestions.

---

Since the whole set of weapons needs a special going over for sanity checking required skills and proficiencies, I think leaving out any skill requirement additions for now makes more sense.

---

> The fail multiplier is similar to the aggregate failure rate from two rounds of crafting, each with a failure rate.
> I might lower it, but I don't think keeping it the same makes sense.

Fair.

> I was on the fence with the saw requirement anyway. I am leaning towards removing it, but welcome suggestions.

I too do not like it, I'm not quite sure is appropriate for the recipes, but I'm no carpenter so I can't really talk, the recipe could maybe use some fine cutting in place of the saw?.

> Since the whole set of weapons needs a special going over for sanity checking required skills and proficiencies, I think leaving out any skill requirement additions for now makes more sense.

Fair, I prefer to do these things even if little by little, but I understand the rationale.